pkg/kubectl/cmd/util/factory.go

```diff
@@ -838,9 +842,9 @@ func (f *Factory) PrinterForMapping(cmd *cobra.Command, mapping *meta.RESTMappin
 		if err != nil {
 			return nil, err
 		}
-		printer = maybeWrapSortingPrinter(cmd, printer)
 	}
 
+	printer = maybeWrapSortingPrinter(cmd, printer, version)
```

Comment:
what does this do if we didn't get in the `ok` block and `version` is undefined?